scipy/io/wavfile.py

```diff
@@ -419,21 +423,42 @@ def _read_data_chunk(fid, format_tag, channels, bit_depth, is_big_endian,
 
     # Number of bytes per sample (sample container size)
     bytes_per_sample = block_align // channels
-    if bit_depth == 8:
-        dtype = 'u1'
-    else:
-        if format_tag == WAVE_FORMAT.PCM:
+
+    if format_tag == WAVE_FORMAT.PCM:
+        if 1 <= bit_depth <= 8 or bytes_per_sample in {3, 5, 6, 7}:
+            dtype = 'u1'
```

Comment:
Actually, instead of adding comments here explaining that `u1` represents raw bytes, it should just use `V1` type.